src/bidiMapper/domains/context/PreloadScriptStorage.ts

```diff
@@ -119,17 +132,40 @@ export class PreloadScriptStorage {
    * Keeps track of the given CDP preload script in the given BiDi preload
    * script.
    */
-  appendPreloadScript(
-    script: BidiPreloadScript,
+  appendCdpPreloadScript(
+    script: BidiPreloadScript, // XXX: id?
```

Comment:
NIT: I don't quite understand the comment. Could you please extend it a bit?

Reply:
The idea was to do this:

```
    let preloadScript: BidiPreloadScript;

    if (typeof script === 'string') {
      preloadScript = this.getPreloadScript({id: script});
    } else {
      preloadScript = script;
    }

    preloadScript.cdpPreloadScripts.push(cdpPreloadScript);
  }
```

But now I realize we don't really need to use it. I'll just remove the comment.